microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -1,70 +1,88 @@
+Detecting and restricting access to your site is not recommended because users may not be able to reach your content or products. However, in some cases you may want to provide different experiences to users of different browsers. For example, if you include steps on how to configure Edge or another browser for use with your site, you may want to detect the browser, and then show the appropriate content to that user.
 
-|  | Server-side | Client-side |  
-|:--- |:--- |:--- | 
-| **User-Agent Client Hints** \(recommended\) | `Sec-CH-UA` HTTPS header | `navigator.userAgentData` JavaScript method |  
-| **User-Agent string** \(legacy\) | `User-Agent` HTTPS header | `navigator.userAgent` JavaScript method |  
+Mechanisms for browser detection with code location and identifiers:
 
-Microsoft recommends that you use [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] whenever possible for the following reasons.  
+| Mechanism | Code Location |  
+|:--- |:--- | 
+| **Feature Detection** \(recommended\) | Client-side: `navigator.userAgentData.brands` JavaScript method |
+| **User-Agent Client Hints** \(recommended\) | Server-side: `Sec-CH-UA` HTTPS header |
+| | Client-side: `navigator.userAgentData` JavaScript method |  
+| **User-Agent String** \(legacy\) | Server-side:`User-Agent` HTTPS header | 
+| | Client-side: `navigator.userAgent` JavaScript method |  
 
-*   Improve code maintainability.  
-*   Reduce code fragility.  
-*   Reduce code breakage from changes to the User-Agent string.  
-    
-When [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] isn't applicable and you must use user agent detection, use the following format to detect Microsoft Edge user agent on Windows and Android.  
+This article describes the methods Microsoft Edge supports for retrieving user agent information.
 
-## User-Agent Client Hints  
+## Feature detection
 
-Starting in Microsoft Edge version 90, access browser information in a cleaner, more privacy-preserving way.  
+Microsoft recommends detecting if a feature is supported in your browser. We do not recommend that you use feature detection to detect a specific browser. For more information see [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection].
 
-### User-Agent Client Hints HTTPS Header  
+If you must detect browsers, we recommend using client hints as follows.
+
+## User Agent Client Hints

```

Comment:
global: User-Agent (w/ hyphen) for consistency.  https://www.bing.com/search?q=%22user+agent+client+hints%22

Reply:
I wasn't sure about this one myself ... most in the topic had no hyphen so I went with that ... welcome other opinions @Reezaali

Reply:
I think either is fine so long as we're consistent

Reply:
Microsoft Writing Style Guide in [Hyphens](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28765) article says "Hyphenate two or more words that precede and modify a noun as a unit if: ... Confusion might result without the hyphen."  

The below spec hyphenates "User-Agent" and uses init-caps; presents it as a formal technical term.  That's a pretty strong indication to follow suit.  Affects What's New 93 too.

Reply:
@Reezaali - pls see spec https://wicg.github.io/ua-client-hints/ which uses "User-Agent", and advise on hyphen & init-caps.

Reply:
Settled on User-Agent